cairo/surface.go

```diff
@@ -39,6 +39,23 @@ func NewSurfaceFromPNG(fileName string) (*Surface, error) {
 	return &Surface{surfaceNative}, nil
 }
 
+// CreateImageSurfaceForData is a wrapper around cairo_image_surface_create_for_data().
+func CreateImageSurfaceForData(data []byte, format Format, width, height, stride int) (*Surface, error) {
+	surfaceNative := C.cairo_image_surface_create_for_data((*C.guchar)(unsafe.Pointer(&data[0])), 
```

Comment:
```suggestion
	surfaceNative := C.cairo_image_surface_create_for_data((*C.uchar)(unsafe.Pointer(&data[0])), 
```
You should use `*C.uchar`. Go 1.9 and lower cannot understand `guchar`, and the "official" signature uses `uchar`.
This will fix the reported build issue for your PR.